src/client-app.js

```diff
@@ -0,0 +1,33 @@
+/* @flow */
+/* eslint-env browser */
+import {compose} from './compose.js';
+import timing, {TimingToken} from './plugins/timing';
+import BaseApp from './base-app';
+import createClientHydrate from './plugins/client-hydrate';
+import createClientRenderer from './plugins/client-renderer';
+
+export default function(): Class<FusionApp> {
+  return class ClientApp extends BaseApp {
+    // TODO: More specific types
+    constructor(element: any, render: (el: any) => Promise<any>) {
+      super();
+      this.register(TimingToken, timing);
+      this.middleware(createClientHydrate(element));
+      this.renderer = createClientRenderer(render);
+    }
+    callback() {
+      this.resolve();
+      const middleware = compose(this.plugins);
+      return () => {
+        // TODO: Create noop context object to match server api
+        const ctx = {
+          url: window.location.path + window.location.search,
+          element: null,
+          body: null,
+        };
+        // $FlowFixMe
```

Comment:
_(minor)_ For flow errors we want to suppress but don't have a plan to 'fix', I'd vote we use `$FlowIgnore` as we do in other plugins.  Helpful for differentiating between what we want fixed and what we want ignored.

Can be enabled by adding the following `.flowconfig`:
```
...
[options]
suppress_comment= \\(.\\|\n\\)*\\$FlowIgnore
...
```